examples/MomentoRedisLambdaExample/cdk.context.json

```diff
@@ -0,0 +1,46 @@
+{
+  "vpc-provider:account=287427698164:filter.vpc-id=vpc-00e94e9613dde8d21:region=us-west-2:returnAsymmetricSubnets=true": {
+    "vpcId": "vpc-00e94e9613dde8d21",
+    "vpcCidrBlock": "172.31.0.0/16",
+    "ownerAccountId": "287427698164",
```

Comment:
Should the account IDs be in the public CDK files?

Reply:
Yes, I thought the same. But, according to their [documentation](https://docs.aws.amazon.com/cdk/v2/guide/context.html#:~:text=Because%20they%27re%20part%20of%20your%20application%27s%20state%2C%20cdk.json%20and%20cdk.context.json%20must%20be%20committed%20to%20source%20control%20along%20with%20the%20rest%20of%20your%20app%27s%20source%20code.%20Otherwise%2C%20deployments%20in%20other%20environments%20(for%20example%2C%20a%20CI%20pipeline)%20might%20produce%20inconsistent%20results.), they suggest to commit cdk.context.json. 

Do you have any suggestions on this?

Reply:
for this use case let's not commit that file, let's put it in gitignore please